src/container_support/serving.py

```diff
@@ -207,8 +207,12 @@ def _invoke(self):
         # Accepting both ContentType and Content-Type headers. ContentType because Coral and Content-Type because,
         # well, it is just the html standard
         input_content_type = request.headers.get('ContentType', request.headers.get('Content-Type', JSON_CONTENT_TYPE))
+
         requested_output_content_type = request.headers.get('Accept') or env.default_accept or JSON_CONTENT_TYPE
 
+        if requested_output_content_type == ANY_CONTENT_TYPE:
+            requested_output_content_type = env.default_accept
```

Comment:
I believe this will end up being None if the environment variable isn't set.

https://github.com/aws/sagemaker-containers/blob/r1.0/src/container_support/environment.py#L318

I believe if the user uses boto3 and doesn't specify an accept type, then boto3 will default to */*.